engine/worker/src/incremental.rs

```diff
@@ -18,7 +18,7 @@ pub fn prepare_incremental_feature_store(
         start_node_id
     );
 
-    let prev_feature_store_uri = setup_job_directory("engine", "feature-store", previous_job_id)
```

Comment:
This might be better addressed in a separate PR, but I’d prefer to avoid having to pass the literal strings "engine" and "workers" every time.

For example, we could:
- create separate functions for the CLI and for workers,
- define constants for "engine" and "worker", or
- use an enum to represent "engine" / "worker".
- etc.

Using literal strings increases the risk of typos, so it would be better to avoid them if possible.
That said, I realize this pattern exists in several other places as well... 😭